examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -254,6 +258,15 @@ void DeviceCallbacks::OnColorControlAttributeChangeCallback(EndpointId endpointI
 }
 #endif
 
+void DeviceCallbacks::OnIdentifyAttributeChangeCallback(EndpointId endpointId, AttributeId attributeId, uint8_t * value)
+{
+        if (attributeId == ZCL_IDENTIFY_TIME_ATTRIBUTE_ID)
+        {
+		uint8_t identifyTime = *value;
+            emberAfWriteServerAttribute(endpointId, ZCL_IDENTIFY_CLUSTER_ID,ZCL_IDENTIFY_TIME_ATTRIBUTE_ID, &identifyTime, ZCL_INT8U_ATTRIBUTE_TYPE);
```

Comment:
I don't understand this line.  We are being notified that the attribute changed.  Our response is to ... write the attribute?  Won't that just end up calling this same code recursively?

What is this PR actually trying to do?